docs/glossary.rst

```diff
@@ -1133,4 +1137,14 @@ Glossary
    Python Packaging Authority
       The `Python Packaging Authority (PyPA) <https://www.pypa.io/en/latest/>`_
       is a working group that maintains many of the relevant projects in Python
-      packaging.
\ No newline at end of file
+      packaging.
+
+   cookiecutter
+      A command-line utility that creates projects from :ref:`cookiecutters <cookiecutter:readme>` (project templates), e.g., creating a Python package project from a Python package project template.
+
+      .. versionadded:: 1.8
+
+      .. seealso:: See also :term:`scaffold`.
```

Comment:
Added in e7b12f6111926cd3073cf1d6365c7a96ef3235b9 and it will be added in the updated `scaffolding.rst` when I get to it.